tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/record/ClosedRecordTypeReferenceTest.java

```diff
@@ -54,20 +54,20 @@ public void setup() {
         compileResult = BCompileUtil.compile("test-src/record/closed_record_type_reference.bal");
     }
 
-    @Test(description = "Negative tests", groups = { "brokenOnNewParser" })
+    @Test(description = "Negative tests" , groups = {"disableOnOldParser"})
     public void negativeTests() {
         CompileResult negative = BCompileUtil.compile("test-src/record/closed_record_type_reference_negative.bal");
         int index = 0;
         BAssertUtil.validateError(negative, index++, "incompatible types: 'PersonObj' is not a record", 28, 6);
         BAssertUtil.validateError(negative, index++, "incompatible types: 'IntOrFloat' is not a record", 35, 6);
         BAssertUtil.validateError(negative, index++, "incompatible types: 'FiniteT' is not a record", 41, 6);
-        BAssertUtil.validateError(negative, index++, "incompatible types: 'int' is not a record", 45, 6);
-        BAssertUtil.validateError(negative, index++, "incompatible types: 'float' is not a record", 46, 6);
-        BAssertUtil.validateError(negative, index++, "incompatible types: 'boolean' is not a record", 47, 6);
-        BAssertUtil.validateError(negative, index++, "incompatible types: 'string' is not a record", 48, 6);
-        BAssertUtil.validateError(negative, index++, "incompatible types: 'byte' is not a record", 49, 6);
-        BAssertUtil.validateError(negative, index++, "incompatible types: 'json' is not a record", 50, 6);
-        BAssertUtil.validateError(negative, index++, "incompatible types: 'xml' is not a record", 51, 6);
+        BAssertUtil.validateError(negative, index++, "only type reference allowed here", 45, 6);
+        BAssertUtil.validateError(negative, index++, "only type reference allowed here", 46, 6);
+        BAssertUtil.validateError(negative, index++, "only type reference allowed here", 47, 6);
+        BAssertUtil.validateError(negative, index++, "only type reference allowed here", 48, 6);
+        BAssertUtil.validateError(negative, index++, "only type reference allowed here", 49, 6);
+        BAssertUtil.validateError(negative, index++, "only type reference allowed here", 50, 6);
+        BAssertUtil.validateError(negative, index++, "only type reference allowed here", 51, 6);
```

Comment:
should it be `only type references are allowed as type inclusions`?

Reply:
I see, I ll change the error message.